Source/Swig/typemap.c

```diff
@@ -2090,14 +2097,22 @@ static void replace_embedded_typemap(String *s, ParmList *parm_sublist, Wrapper
 	  Printf(stdout, "Swig_typemap_attach_parms:  embedded\n");
 #endif
 	  if (already_substituting < 10) {
+	    char* found_colon;
 	    already_substituting++;
 	    if ((in_typemap_search_multi == 0) && typemap_search_debug) {
 	      String *dtypemap = NewString(dollar_typemap);
 	      Replaceall(dtypemap, "$TYPEMAP", "$typemap");
 	      Printf(stdout, "  Containing: %s\n", dtypemap);
 	      Delete(dtypemap);
 	    }
-	    Swig_typemap_attach_parms(tmap_method, to_match_parms, f);
+	    found_colon = Strstr(tmap_method, ":");
```

Comment:
`Strchr(tmap_method, ':')` would be better since we only want to find a single character.